libass/ass_types.h

```diff
@@ -274,7 +274,7 @@ typedef struct ass_track {
     int WrapStyle;
     int ScaledBorderAndShadow; // 0 or 1 (boolean)
     int Kerning; // 0 or 1 (boolean)
-    char *Language; // zero-terminated ISO-639-1 code
+    char *Language; // if non-null, zero-terminated BCP47 code including language and optional region
```

Comment:
I feel like this can be misread as requiring not only the the – usually optional – region part to be included in `Language`’s BCP47 code.
Due to its importance for interoperability with ISO-639-1-only interfaces, it may also be worth to reiterate BCP47’s requirement to use 2letter codes when available.